tests/sentry/tasks/test_post_process.py

```diff
@@ -1862,3 +1863,11 @@ def test_occurrence_deduping(self, mock_processor):
 
         # Make sure we haven't called this again, since we should exit early.
         assert mock_processor.call_count == 1
+
```

Comment:
These tests started breaking because the `eventstore.get` interface of redis is stricter than default. we didn't actually regress, the test just nooped this entire time:

```
# This works under the default eventstore, and is broken in both prod
# and the new default eventstore
assert eventstore.get(None) is None
```